srml/system/Cargo.toml

```diff
@@ -13,6 +13,7 @@ rstd = { package = "sr-std", path = "../../core/sr-std", default-features = fals
 runtime_io = { package = "sr-io", path = "../../core/sr-io", default-features = false }
 primitives = { package = "sr-primitives", path = "../../core/sr-primitives", default-features = false }
 srml-support = { path = "../support", default-features = false }
+curve25519-dalek = { version = "1.2.1", default-features = false, features = ["u64_backend"] }
```

Comment:
Why do we need this? (Is not used yet)

Reply:
I was dealing with compile-time errors.  I will see if I can remove this.

Reply:
Done.